### DISCUSSION TITLE
"survey:date" instead of a "check_date" in "Check if element still exists"

### DISCUSSION THREAD

**Discussion by atarom** [streetcomplete/StreetComplete#4108] [Feedback]
Hello, I am quite new to all these things, if when performing a "Check if element still exists"
the source=survey tag is generated in the changeset; is the  "survey:date" instead of a "check_date" in the tags of the reviewed element better?
survey:date -> ... to indicate the date when a node, way or relation was last surveyed or verified in person ...
check_date -> ... similar but more general tag, does not specify that the information was checked by survey ...
https://wiki.openstreetmap.org/wiki/Key:survey:date
Cheers

**Comment by HolgerJeromin** [comment]
Thanks for you thoughts:

See last discussion on that topic:
https://github.com/streetcomplete/StreetComplete/issues/2592#issuecomment-782918975
